### ISSUE TITLE
[fzf] Pass different options to fzf for display

### ISSUE THREAD

**Issue by runofthemillgeek** [SidOfc/cani#12]
Hi there! Loving the tool you created. I have `FZF_DEFAULT_OPTS` set in my `.zshrc` which sets preview window by default. As you may have guessed, this doesn't really work well with cani. Is it possible to pass or set different fzf options for cani?

**Comment by SidOfc** [comment]
Hey there, thanks for reporting this issue!

I don't think I made it configurable but I can figure out a way to make it configurable at least, in that way the user has control over the options passed, I think a nice place to put it would be in the configuration file that already gets created upon initialization.

Right now I'm heading to bed (2AM here, got work tomorrow) but I'll work on it tomorrow and hopefully release a new version on the same day :+1:

**Comment by SidOfc** [comment]
Hi @sangeeth96!

I went with a much simpler solution that just disables the preview by applying `--no-preview` explicitly. I also found another bug that would make `cani` crash when being run without arguments so cheers for leading me to that discovery as well :+1: 

Issues should be fixed and a new version of `cani` has been released. Run: `gem update cani` and see if the version you install is indeed `0.5.6`.

Let me know if you experience any other issues :v:

**Comment by runofthemillgeek** [comment]
@SidOfc Yaay! Seems to be working alright now:

![image](https://user-images.githubusercontent.com/2759499/61864410-0113c380-aeef-11e9-812d-42e96d56ac61.png)

May I ask if it's supposed to take only half the width of the terminal or is that also because of my `FZF_DEFAULT_OPTS`?

**Comment by SidOfc** [comment]
The reason it does not span the entire terminal is because all data is completely shown and nothing else comes after it, this is how FZF behaves by default. If you'd like to see more information check out the config file located in `~/.config/cani/config.yml` which lists the default browsers shown by `cani` in a `browsers` key.

The values you can use are the _keys_ of the _agents_ object from Caniuse JSON data which can be found here: https://raw.githubusercontent.com/Fyrd/caniuse/master/data.json.

The `browsers` key also defines the order in which the browsers are displayed and can be safely changed to match your personal preference as well :)

**Comment by SidOfc** [comment]
Hi @sangeeth96!, I think I misunderstood when you first asked about the screen width taken up by FZF. I truncated the titles to 24 characters max. I've added a check and published version `0.5.8` on rubygems which allow titles to grow up to 50 characters in the `cani use` screen.

This also happens to make features more searchable since more of the text is available to actually search for now.

![image](https://user-images.githubusercontent.com/3225058/62012170-eeb1b800-b182-11e9-962d-55b44e5c03d9.png)
